src/main/scala/mesosphere/marathon/storage/repository/GcActor.scala

```diff
@@ -69,7 +70,8 @@ private[storage] class GcActor[K, C, S](
     val appRepository: AppRepositoryImpl[K, C, S],
     val podRepository: PodRepositoryImpl[K, C, S],
     val maxVersions: Int,
-    val scanBatchSize: Int = 32)(implicit val mat: Materializer, val ctx: ExecutionContext)
+    val scanBatchSize: Int = 32,
+    val zkCleaningInterval: FiniteDuration)(implicit val mat: Materializer, val ctx: ExecutionContext)
```

Comment:
nit: s/zkCleaningInterval/cleaningInteveral (you don't have `zk` prefix for e.g. scanBatchSize, do you?)